docs/docs/examples/src/layouts/bar/image.js

```diff
@@ -0,0 +1,10 @@
+var module = pathfora.Message({
+  id: 'bar-image',
+  layout: 'bar',
+  headline: 'Welcome',
```

Comment:
the bar does not support headline so we can take this line out as to not confuse